packages/cli/src/__tests__/lib/commands/scenes/scenes-util.test.ts

```diff
@@ -0,0 +1,36 @@
+import { selectFromList } from '@smartthings/cli-lib'
+import ScenesCommand from '../../../../commands/scenes'
+import { chooseScene } from '../../../../lib/commands/scenes/scenes-util'
+import { ScenesEndpoint } from '@smartthings/core-sdk'
+import { Config } from '@oclif/core'
+
+
+const listSpy = jest.spyOn(ScenesEndpoint.prototype, 'list').mockImplementation()
+
+describe('chooseScene', () => {
+	const mockSelectFromList = jest.mocked(selectFromList)
+
+	it('calls selectFromList with correct config and endpoint', async () => {
+		const command = new ScenesCommand([], new Config({ root: '' }))
+
+		mockSelectFromList.mockImplementationOnce(async (_command, _config, options) => {
+			await options.listItems()
+			return 'selected-scene-id'
+		})
+
+		await chooseScene(command)
+
+		expect(selectFromList).toBeCalledWith(
+			command,
+			expect.objectContaining({
+				itemName: 'scene',
+				primaryKeyName: 'sceneId',
+				sortKeyName: 'sceneName',
+			}),
+			expect.objectContaining({ preselectedId: undefined }),
+		)
+
+		expect(listSpy).toBeCalledTimes(1)
+	})
+})
+
```

Comment:
This particular example feels a little out-of-scope testing-wise to me because the `listItems` method that's passed is the endpoint's list method.  I think it'd be great to do for others, though, and will keep that in mind for future PRs.